fdbserver/include/fdbserver/DataDistributionTeam.h

```diff
@@ -22,6 +22,35 @@
 
 #include "fdbclient/StorageServerInterface.h"
 
+struct GetTeamRequest;
+namespace data_distribution {
```

Comment:
Because I think the eligibility type and this counter only make sense in the context of data distribution, and this is a header file that can be included in everywhere. So I did it for semantic clarity.